### PR TITLE
Allow the mail_from address to be overridden

### PR DESCRIPTION
**Pull request by fessyfoo**

### CODE REVIEW

File: files/mailer.rb [hunk 1]
```diff
@@ -54,12 +54,12 @@ def mail_destination
...
 
   def handle
-    
+
     mail_to = mail_destination
     # Only procede if we have an email address to work with
     return false unless mail_to
 
-    mail_from = 'sensu@yelp.com'
+    mail_from = settings['mailer']['mail_from'] || 'sensu@yelp.com'
```
Comment (somic) [inline files/mailer.rb]: Maybe make it handler_settings['mail_from'] like other settings, for consistency?

Comment (fessyfoo) [inline files/mailer.rb]: agreed.

**Comment by solarkennedy** [comment]
Can you remove the yelp references entirely here? Just document the parameter and I'll take care of making sure I inject the right config on our side after merge.

**Comment by fessyfoo** [comment]
references removed.

**Comment by fessyfoo** [comment]
is this blocked on anything from me?

**Comment by solarkennedy** [comment]
Thanks! I need to remind myself to merge this into our codebase with a new hiera setting.

**Pull request merged**